src/styles/_catalog.scss

```
@@ -1,22 +1,44 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
 
   max-width: 944px;
   margin: 0 auto;
+  gap: 48px 48px;
+
+  justify-content: center;
+
+  grid-template-columns: repeat(auto-fill, minmax(200px, 200px));
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
+    display: grid;
+    place-items: center;
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 487px) {
+    & {
+      grid-template-columns: repeat(1fr);
+    }
+  }
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(1fr);
+    }
```

```suggestion
    grid-template-columns: repeat(2, 1fr);
```

No need to use & symbol if we are applying styles to general class